copier/user_data.py

```diff
@@ -173,6 +177,7 @@ class Question:
     answers: AnswersMap
     jinja_env: SandboxedEnvironment
     choices: Union[Sequence[Any], Dict[Any, Any]] = field(default_factory=list)
+    multiple: bool = False
```

Comment:
**issue:** the `multiple` word might seem OK as of today, but when https://github.com/copier-org/copier/issues/750 is implemented some day, it could become confusing.

This PR is about having the possibility to select multiple choices in a choices question. However, https://github.com/copier-org/copier/issues/750 is about asking the same question multiple times. In other words, the issue you're fixing is https://github.com/copier-org/copier/issues/218 instead.

Looking at https://github.com/RhetTbull/questionary-superprompt#multiple-option, it's possible that we use that solution when we implement https://github.com/copier-org/copier/issues/750. Thus, I'm afraid that using `multiple` now could be a shot in our own foot for the future.

So I think we should be able to rename this now, to help our future self.

WDYT about renaming this to `multiselect`?


```suggestion
    multiselect: bool = False
```

The same for other occurrences of this option.

FWIW, I can imagine a questionary where somebody wants to be able to make a multiselect choice multiple times (in the future, after this and #750 are implemented).

Reply:
Oh, I am OK with this, I chose `multiple` as this was the term chosen in #750 (#750 define 2 behaviors, `multiple: true` and `multiple: object`) and #1049.

In fact, I thought a lot about it (that's what I was implying when saying "the advanced form needs more specification and design I think") because I think part of questionnary-superprompt might also be useful for multiselect: min/max make sense in multiselect as much as it make sense in multiple prompts. Poitn being that a possible evolution of `multiselect: true` is the same pattern than `multiple: true`: allowing to defined expected boundaries.

So `multiselect` it is, I'll update the PR accordingly 👍🏼 (thanks for pointing #218 because I missed it)